<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="3" hostname="DESKTOP-LPSVTKP" failures="1" tests="2" name="Default test(failed)" time="0.11" errors="0" timestamp="2022-12-17T10:58:22 IST">
  <testcase classname="failed_Xml.Failed_XmlEx3" name="A" time="0.018">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at failed_Xml.Failed_XmlEx3.A(Failed_XmlEx3.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- A -->
  <testcase classname="failed_Xml.Failed_XmlEx3" name="C" time="0.004"/>
  <testcase classname="failed_Xml.Failed_XmlEx3" name="B" time="0.0">
    <ignored/>
  </testcase> <!-- B -->
  <testcase classname="failed_Xml.Failed_XmlEx3" name="E" time="0.0">
    <ignored/>
  </testcase> <!-- E -->
  <testcase classname="failed_Xml.Failed_XmlEx3" name="D" time="0.0">
    <ignored/>
  </testcase> <!-- D -->
</testsuite> <!-- Default test(failed) -->
